packages/react-charting/src/utilities/ChartHoverCard/ChartHoverCard.tsx

```diff
@@ -10,3 +10,5 @@ export const ChartHoverCard: React.FunctionComponent<IChartHoverCardProps> = sty
   IChartHoverCardStyleProps,
   IChartHoverCardStyles
 >(ChartHoverCardBase, getChartHoverCardStyles);
+
+export default ChartHoverCard;
```

Comment:
Is this needed?

Reply:
Yes. Lazy load can only import default exports.
https://dev.to/iamandrewluca/react-lazy-without-default-export-4b65

Reply:
Can't we use this method:
() => import('./OtherComponent').then(module => ({ default: module.OtherComponent }))

Reply:
Done